packages/allocator/Dockerfile

```diff
@@ -29,8 +29,10 @@ WORKDIR /app
 COPY pg_hba.conf /etc/postgresql/15/main/pg_hba.conf
 
 # Install package from PyPI using UV in explicit venv location
-RUN uv venv /app/.venv && \
-    uv pip install --python=/app/.venv/bin/python lablink-allocator-service==${PACKAGE_VERSION}
+RUN uv init --python 3.11 --no-readme && \
```

Comment:
we can commit the python version if you prefer that, i believe that is supposed to be best practice https://docs.astral.sh/uv/concepts/python-versions/#python-version-files